pkg/runtime/authorizer/component.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package authorizer
+
+import (
+	"strings"
+
+	componentsV1alpha1 "github.com/dapr/dapr/pkg/apis/components/v1alpha1"
+)
+
+type componentDenyList struct {
+	list []componentDenyListItem
+}
+
+func newComponentDenyList(raw []string) componentDenyList {
+	list := make([]componentDenyListItem, len(raw))
+	i := 0
+	for _, comp := range raw {
+		if comp == "" {
+			continue
+		}
+		v := strings.Split(comp, "/")
```

Comment:
Please comment the format here for better understanding.

Reply:
I did not author this code, and has been copied and pasted from [here](https://github.com/dapr/dapr/blob/master/pkg/runtime/component_authorizer.go). This is done because it needs to be available to the hot reloading reconciler. We can address this file in a follow-up PR.

Reply:
OK, let's create an issue to track it.